bookmarks.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Quiz-App</h1>
+    </header>
+    <main>
+      <ul class="card-list">
+        <li class="question-card">
+          <a href="#"
+            ><svg
+              class="bookmark"
+              xmlns="http://www.w3.org/2000/svg"
+              class="icon icon-tabler icon-tabler-bookmark"
+              width="3em"
+              height="3em"
+              viewBox="0 0 24 24"
+              stroke-width="2"
+              stroke="black"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              fill="black"
+            >
+              <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+              <path
+                d="M9 4h6a2 2 0 0 1 2 2v14l-5 -3l-5 3v-14a2 2 0 0 1 2 -2"
+              ></path>
+            </svg>
+          </a>
+          <h2 class="question">
+            What is Flexbox and what problem does it solve?
+          </h2>
+          <button type="button" class="answer-button">
+            <p class="button-text">Show Answer</p>
+          </button>
+          <ul class="tag-box-list">
+            <li class="tag-box-item">#html</li>
+            <li class="tag-box-item">#flexbox</li>
+            <li class="tag-box-item">#css</li>
+          </ul>
+        </li>
+      </ul>
```

Comment:
you are using more `ul `s than necesssary here

you can put more `li`in one `ul`